cloudslang-content-verifier/pom.xml

```diff
@@ -18,6 +18,12 @@
         </dependency>
 
         <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>cloudslang-entities</artifactId>
```

Comment:
<ignoredUsedUndeclaredDependency>io.cloudslang.lang:cloudslang-entities:jar:${project.version}</ignoredUsedUndeclaredDependency> should be removed from this module